### ISSUE TITLE
Add support for sending keystrokes or commands to multiple panes

### ISSUE THREAD

**Issue by Riari** [microsoft/terminal#7932]
# Description of the new feature/enhancement

With the current implementation of panes, it's not possible to send keystrokes or commands to multiple panes at once; you have to navigate to each pane to interact with them one by one. It would be nice if we could do one of the following:

A) Send keys to specific panes (a la tmux's `send-keys`)
B) Focus multiple panes at once in order to interact with them simultaneously
C) Open a text input via a new keyboard shortcut to enter text and hit enter to send it to all panes in the current tab

B or C would be preferable for my use cases (e.g. checking out a specific branch across multiple git repos), but I think any of these would be a welcome addition.

**Comment by Don-Vito** [comment]
Seems to duplicate https://github.com/microsoft/terminal/issues/2634

**Comment by zadjii-msft** [comment]
Yep, this is definitely /dup #2634. Was there a twitter thread about this recently or something? #7919 was also just recently filed about this same thing 🤔
